File: language/vowelDeleter.py
words = ['Slate', 'Earthy', 'Quick']
output = ''
for word in words:
    word = word.lower()
    tempWord = ''
    for letter in word:
        if letter != 'a' and letter != 'e' and letter != 'i' and letter != 'o' and letter != 'u' and letter != '':
            tempWord = tempWord.join([letter, ''])    
    output = output.join([tempWord, ''])
    output = output.join([' ', ''])
output = output[::-1]
print(output)

File: language/vowelDeleter.js
const words = ['Slate', 'Earthy', 'Quick'];
let output = '';
for (var word of words) {
  word = word.toLowerCase();
  let tempWord = '';
  for (const letter of word) {
    if (!['a', 'e', 'i', 'o', 'u'].includes(letter)) {
      tempWord = tempWord.concat(letter, '');
    }
  }
  output = output.concat(tempWord, ' ');
}
console.log(output);
